app/javascript/components/gtl/DataTable.jsx

```diff
@@ -30,79 +29,91 @@ export const DataTable = ({
   onPerPageSelect,
   onPageSet,
 }) => {
-
   const selectAll = () => {
-    const [checkedItems, setCheckedItems] = useState({}); //plain object as state
+    const [checkedItems, setCheckedItems] = useState({}); // plain object as state
 
     const localOnSelectAll = (ev) => {
-      if (ev.target.classList.contains('is-checkbox-cell') ||
-        ev.target.parentElement.classList.contains('is-checkbox-cell')) {
+      if (ev.target.classList.contains('is-checkbox-cell')
+        || ev.target.parentElement.classList.contains('is-checkbox-cell')) {
         return;
       }
 
       onSelectAll(rows, ev.target);
-      setCheckedItems({...checkedItems, [ev.target.name]: ev.target.checked});
+      setCheckedItems({ ...checkedItems, [ev.target.name]: ev.target.checked });
       ev.stopPropagation();
     };
 
     return (
-      <input type="checkbox"
-             name={'selectAll'}
-             key={'selectAll'}
-             checked={!!checkedItems['selectAll']}
-             onChange={localOnSelectAll}
-             onKeyPress={localOnSelectAll}
-             role="checkbox"
-             aria-checked="false"
-             tabIndex="0"
-             aria-labelledby="selectAll"
-             title="Check All"/>
+      <input
+        type="checkbox"
+        name="selectAll"
+        key="selectAll"
+        checked={!!checkedItems.selectAll}
+        onChange={localOnSelectAll}
+        onKeyPress={localOnSelectAll}
+        role="checkbox"
+        aria-checked="false"
+        tabIndex="0"
+        aria-labelledby="selectAll"
+        title="Check All"
+      />
     );
   };
 
   const renderTableHeader = () => (
     <thead className="miq-thead">
       <tr>
-        {!inEditMode() && !noCheckboxes() &&
-        <th className="narrow table-view-pf-select">
-          <label className="hiddenCheckboxLabel" id="selectAll" aria-hidden="true">{__('Select All')}</label>
-          {selectAll()}
-        </th>
-        }
+        {!inEditMode() && !noCheckboxes()
+        && (
+          <th className="narrow table-view-pf-select">
+            <label className="hiddenCheckboxLabel" id="selectAll" aria-hidden="true">{__('Select All')}</label>
+            {selectAll()}
+          </th>
+        )}
```

Comment:
@kavyanekkalapu Are you using some sort of linting tool that is rearranging the code layout?  If so, is it possible to save those layout changes as a separate commit (or even PR) from the logic changes?  Otherwise, it's very difficult to review because I can't tell what's actually changed.

(Even better, if we can just blast out linter changes to every file and save that, then we won't have this problem).

Reply:
Yes that file has lot of linting issues, i fixed that, will revert those changes and can make new pr.